src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerLivenessProbe.java

```diff
@@ -96,5 +100,69 @@ public String toString() {
         public String getDisplayName() {
             return "Container Liveness Probe";
         }
+
+        public static final String DEFAULT_EXEC_ARGS = "uptime";
```

Comment:
I don't see the point of adding a liveness probe to to be honest. The default is just to run some sleep command, so why would you need for a liveness probe?

Reply:
I'm going to try to make this work with empty values in the form, I don't think we want to have a default liveness probe. The snippet generator doesn't like not having defaults though.

Reply:
Solved by changing the jelly from <f:entry> to <f:optionalProperty>